src/FINDOLOGIC/Export/Helpers/UsergroupAwareMultiValueItem.php

```diff
@@ -19,7 +19,7 @@
 
     public function __construct($itemName, $value, $usergroup)
     {
-        $this->value = $value;
+        $this->value = DataHelper::emptyValueCheck($value);
```

Comment:
[Suppress](https://phpmd.org/documentation/suppress-warnings.html) the Code Climate warning for using static access here, because this is a legitimate use case.

Reply:
Done